test/unit/vl-multiselect.test.html

```diff
@@ -36,17 +46,35 @@
       });
 
       test('wanneer het vl-multiselect element volledig klaar is zal het een ready event versturen', (done) => {
-        const element1 = fixture('vl-multiselect-fixture');
-        element1.addEventListener(element1.readyEvent, () => {
-          const element2 = fixture('vl-multiselect-auto-dress-fixture');
-          element2.addEventListener(element2.readyEvent, () => {
+        const element = fixture('vl-multiselect-fixture');
+        element.addEventListener(element.readyEvent, () => {
+          done();
+        });
+        element.dress();
+      });
+
+      test('wanneer het vl-multiselect element success of error attribuut bevat zal dit zichtbaar zijn', (done) => {
+        let element = fixture('vl-multiselect-fixture-success');
+        element.addEventListener(element.readyEvent, async () => {
+          await awaitUntil(() => {
+            const container = element.parentElement.parentElement.parentElement;
```

Comment:
kunnen we hier geen selector pakken ipv dit te moeten doen?

Reply:
Alleen het element (select tag) gebruiken wij, de (dressed) containers worden gegenereerd en bevatten geen id, daarom doe ik de selector liever via parent element dan op classes die morgen kunnen wijzigen.